mbuild/coordinate_transform.py

```diff
@@ -434,6 +434,26 @@ def translate_to(compound, pos):
     _set_particle_positions(compound, atom_positions)
 
 
+def rotate(compound, theta, rotate_around):
```

Comment:
How about naming the keyword `around`? Makes for cleaner syntax IMO and reads more naturally:

`mb.rotate(ethane, 180, around=[1, 0, 0])`

`rotate ethane 180 degrees around the vector [1, 0, 0]`